patients/repo.go

```diff
@@ -95,6 +95,7 @@ func (r *repository) Initialize(ctx context.Context) error {
 		{
 			Keys: bson.D{
 				{Key: "clinicId", Value: 1},
+				{Key: "userId", Value: 1},
```

Comment:
The creation of this index will fail because indexes cannot be updated - you should use a different name.